src/Field/WfsSearch/WfsSearch.jsx

```diff
@@ -0,0 +1,389 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Select,
+  Spin
+} from 'antd';
+const Option = Select.Option;
+
+import Logger from '../../Util/Logger';
+
+import OlMap from 'ol/map';
+import OlFormatFilter from 'ol/format/filter';
+import OlFormatGeoJSON from 'ol/format/geojson';
+import OlFormatWFS from 'ol/format/wfs';
+
+/**
+ * The WfsSearch field.
+ * Implements an input field to do a WFS-GetFeature request.
+ *
+ * The GetFeature request is created with `ol.format.WFS.writeGetFeature`
+ * so most of the WFS specific options work like document in the corresponding
+ * API-docs: http://openlayers.org/en/latest/apidoc/ol.format.WFS.html#writeGetFeature
+ *
+ * @class WfsSearch
+ * @extends React.Component
+ */
+export class WfsSearch extends React.Component {
+
+  /**
+   * The className added to this component.
+   * @type {String}
+   * @private
+   */
+  className = 'react-geo-wfssearch'
+
+  static propTypes = {
+    /**
+     * A optional className that will be added to the input field.
+     */
+    className: PropTypes.string,
+    /**
+     * The base URL.
+     * @type {String}
+     */
+    baseUrl: PropTypes.string.isRequired,
+    /**
+     * The Property which should be shown in the inputfield when a selection was
+     * made.
+     * @type {String}
+     */
+    displayField: PropTypes.string,
+    /**
+     * The list of attributes that should be searched through.
+     * @type {String[]}
+     */
+    searchAttributes: PropTypes.arrayOf(PropTypes.string).isRequired,
+    /**
+     * The namespace URI used for features.
+     * @type {String}
+     */
+    featureNS: PropTypes.string,
+    /**
+     * The prefix for the feature namespace.
+     * @type {String}
+     */
+    featurePrefix: PropTypes.string,
+    /**
+     * The feature type names. Required.
+     * @type {String[]}
+     */
+    featureTypes: PropTypes.arrayOf(PropTypes.string).isRequired,
+    /**
+     * SRS name. No srsName attribute will be set on geometries when this is not
+     * provided.
+     * @type {string}
+     */
+    srsName: PropTypes.string,
+    /**
+     * The output format of the response.
+     * @type {string}
+     */
+    outputFormat: PropTypes.string,
+    /**
+     * Maximum number of features to fetch.
+     * @type {number}
+     */
+    maxFeatures: PropTypes.number,
+    /**
+     * Geometry name to use in a BBOX filter.
+     * @type {string}
+     */
+    geometryName: PropTypes.string,
```

Comment:
A utility to get this from a DescribeFeatureType would be nice

Reply:
Sounds very useful. Can you open an issue for it.

Reply:
Here you are: #567